app/static/config.js

```diff
@@ -1,7 +1,7 @@
 const config = {
```

Comment:
this config file will not be needed as we'll just import everything static assets after we move all avatars to our project